istio.deps

```diff
@@ -11,6 +11,6 @@
 		"name": "ENVOY_SHA",
 		"repoName": "envoyproxy/envoy",
 		"file": "WORKSPACE",
-		"lastStableSHA": "346059548e135199eb0b7f0006f3ef19e173bf79"
```

Comment:
you also need to change the one in WORKSPACE

Reply:
thanks, done